en/core-libraries/inflector.rst

```diff
@@ -80,6 +81,9 @@ table names or class names::
     // UserProfileSetting
     Inflector::classify('user_profile_settings');
```

Comment:
I don't think I changed Classify... thats from the old docs. If that changed from 2.x it should be in the migration docs?